server/server.js

```diff
@@ -218,6 +219,30 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
   }
 };
 
+// Process the resource messages associated with a single client
+Server.prototype._handleResourceMessages = function (socket, client) {
+  var subscriptions = client.subscriptions;
+  var presences = client.presences;
+
+  // Pause events on the inbound socket
+  Pauseable.pause(socket);
+
+  for (var key in subscriptions) {
+    var message = subscriptions[key];
```

Comment:
Yep, I prefer this also.  There was a mixture of old and new styles in this file, so I standardized to this form.